dotcom-rendering/src/components/Discussion.tsx

```diff
@@ -172,13 +192,6 @@ export const Discussion = ({
 		}
 	}, []);
 
-	useEffect(() => {
-		// There's no point showing the view more button if there isn't much more to view
-		if (commentCount === 0 || commentCount === 1 || commentCount === 2) {
-			setIsExpanded(true);
-		}
-	}, [commentCount]);
-
```

Comment:
I think that the pattern of having state being derived from other state is pretty sound. Why fold this into `getDiscussion`’s then?

Reply:
Of course, that makes sense, thanks for clarifying!